pages/common/ical.md

```diff
@@ -1,6 +1,6 @@
 # ical
 
-> A Hirji/Islamic calendar and converter for the terminal.
+> Convert Hirji/Islamic calendars.
```

Comment:
```suggestion
> Convert and view Hirji and Islamic calendars.
```
> This doesn't look right.

@cyqsimon Why not? I support changing this for quite a few reasons:
- Obviously – and probably not controversially (it's on the style guide since a long time) – we don't need the in your terminal part because tldr documents CLI commands that often have a terminal-only interface (TUI) / output. We just to make it clear when the command open a GUI interface.
- This change fits the style guide's guideliene of starting with an imperative verb.
- In some other PRs, I've changed descriptions with "converters" to "convert", so I don't think this is controversial, on the contrary, it improves the consistency among the pages.

However, I definitely see some improvement points:
- In this context, "and" is better than "/", right?
- It also allow the user to view the calendars, I overlooked this

But I was a bit confused when you said that it doesn't feel right, you know? :sweat_smile:

Reply:
Yeah sorry I made my comment rather quickly and didn't explain. My bad.

Yeah my primary issue was the missing "viewing" part. And if we are particularly nitpicking, I'd also argue that you don't "convert a calendar", rather you "convert dates in a calendar", but I recognise that's difficult to express concisely. Up to you.